### ISSUE TITLE
Context aware variable existence detection fails on inline functions

### ISSUE THREAD

**Issue by SkaceKamen** [SkaceKamen/vscode-sqflint#37]
Context aware detection fails under cenrtain conditions
```sqf
_array = [];
_something = 1;

// _something is reported undefined
_array select { _x == _something }; 

// _something is reported undefined
_array findIf { _x == _something };

// _something, _array is reported undefined in the inner blocks
(count(_array) == 1 && { count(_array) == 1 && { _someting == 1 && count(_array) == 0 } });
```

(moved from #36)

**Comment by SkaceKamen** [comment]
Fixed in 0.9.5
